google-cloud-speech/README.md

```diff
@@ -26,7 +26,7 @@ require "google/cloud/speech"
 speech = Google::Cloud::Speech.new
 
 audio = speech.audio "path/to/audio.raw",
```

Comment:
raw and wav are both file formats, but a raw file can include various types of encoded audio.

So I think raw is correct.

That said... using the Speech API introduced me to the .raw format, which I didn't previously know.  .wav, however, is something that "everybody knows"...